image_client.py

```diff
@@ -126,10 +126,12 @@ def upload_to_image_server(self, full_path, redacted, collection, original_path=
             r = requests.get(self.build_url("getfileref"), params=params)
             url = r.text
             assert r.status_code == 200
+            logging.info(f"Uploaded: {local_filename},{attach_loc},{url}")
```

Comment:
if it succeeds with a 200 code, the path is simply added to the .txt file list with no flag.